suites/pacific/rgw/tier_0_rgw.yaml

```diff
@@ -119,8 +119,3 @@ tests:
       module: sanity_rgw.py
       name: Swift based tests
       polarion-id: CEPH-11019
-
-  - test:
-      name: generate sosreport
-      module: sosreport.py
-      desc: generate sosreport
```

Comment:
For pacific, we would need have the test case for functional hence don't remove from here...

Reply:
Made changes